### DISCUSSION TITLE
How can I suspend user or admin account?

### DISCUSSION THREAD

**Discussion by ilshm** [pocketbase/pocketbase#4396] [Q&A] [answered]
Is there a method to deactivate a user or admin account?

I've experimented with hooks, but they only prevent new logins and token renewals.

What is a clean and straightforward approach to restrict a user's access?

Thank you.

**Comment by ganigeorgiev** [comment, marked as answer]
Users (or admins) don't have "active" or "disabled" status in PocketBase (at least not out of the box).

For the regular users you can create your own custom field (ex. `status`) and check its value in the API rules and/or perform the check via the hooks (to reset previous tokens you can invoke `record.RefreshTokenKey()` and save the record).

For the admins you can safely delete the entry as admins don't have any associations and all admins have the same permissions.

**Reply by ilshm** [reply]
Thanks, a bit complex, but hope will be added in the future.
